desktop/cmp/dash/canvas/impl/DashCanvasContextMenu.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+
+import {hoistCmp} from '@xh/hoist/core';
+import {contextMenu} from '@xh/hoist/desktop/cmp/contextmenu';
+import {Icon} from '@xh/hoist/icon';
+import {isEmpty} from 'lodash';
+
+/**
+ * Context menu to add items to a DashCanvas
+ *
+ * -------- !! NOTE: THIS COMPONENT IS CURRENTLY IN BETA !! --------
```

Comment:
Beta warning not needed here --- actually, just want on the main public entry points. - Component and model